hiera/hieradata/dev/wso2/wso2as/5.3.0/default.yaml

```diff
@@ -0,0 +1,94 @@
+---
+classes:
+  - wso2as
+wso2::pack_filename: "%{::product_name}-%{::product_version}.zip"
+wso2::pack_extracted_dir: "%{::product_name}-%{::product_version}"
+
+wso2::template_list :
+  - repository/conf/axis2/axis2.xml
+  - repository/conf/carbon.xml
+  - repository/conf/registry.xml
+  - repository/conf/datasources/master-datasources.xml
+  - repository/conf/tomcat/catalina-server.xml
+  - bin/wso2server.sh
+
+wso2::service_name: wso2as
+wso2::service_template: wso2base/wso2service.erb
+wso2::file_list: []
+
+wso2::clustering :
+    enable : true
+    local_member_host : 127.0.0.1
+    local_member_port : 4000
+    membership_scheme : wka
+    wka :
+       members :
+         -
+           hostname : localhost
+           port : 4000
+         -
+           hostname : localhost
+           port : 4001
+    multicast :
+       domain : wso2.carbon.domain
+    k8s :
+    ppaas :
+
+wso2::ports:
+    proxyPort :
+      http: 9763
+      https: 9443
+
+wso2::usermgt_datasource: WSO2USER_DB
+
+wso2::master_datasources :
+  WSO2_CONFIG_DB:
```

Comment:
@ravihansa3000 The only thing I'm concerned is using an uppercase text for an element name in a YAML file. Ideally we should use YAML naming convention for this.